src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,16 +1,126 @@
 """Wrappers for py_bullet environments."""
 import inspect
 
+import akro
+import gym
 from pybullet_envs.bullet.minitaur_duck_gym_env import MinitaurBulletDuckEnv
 from pybullet_envs.bullet.minitaur_gym_env import MinitaurBulletEnv
 from pybullet_envs.env_bases import MJCFBaseBulletEnv
 
-from garage.envs import GarageEnv
+from garage.envs.env_spec import EnvSpec
 
 
-class BulletEnv(GarageEnv):
+class BulletEnv(gym.Wrapper):
     """Binding for py_bullet environments."""
 
+    def __init__(self, env=None, env_name='', is_image=False):
+        """Returns a Garage wrapper class for bullet-based gym.Env.
+
+        Args:
+            env (gym.wrappers.time_limit): A gym.wrappers.time_limit.TimeLimit
+                object wrapping a gym.Env created via gym.make().
+            env_name (str): If the env_name is speficied, a gym environment
+                with that name will be created. If such an environment does not
+                exist, a `gym.error` is thrown.
+            is_image (bool): True if observations contain pixel values,
+                false otherwise. Setting this to true converts a gym.Spaces.Box
+                obs space to an akro.Image and normalizes pixel values.
+
+        """
+        if not env:
+            # 'RacecarZedBulletEnv-v0' environment enables rendering by
+            # default, while pybullet allows only one GUI connection at a time.
+            # Setting renders to False avoids potential error when multiple
+            # of these envs are tested at the same time.
+            if env_name == 'RacecarZedBulletEnv-v0':
+                env = gym.make(env_name, renders=False)
+            else:
+                env = gym.make(env_name)
+
+        # Needed for deserialization
+        self._env = env
+        self._env_name = env_name
+
+        super().__init__(env)
+        self.action_space = akro.from_gym(self.env.action_space)
+        self.observation_space = akro.from_gym(self.env.observation_space,
+                                               is_image=is_image)
+        self.__spec = EnvSpec(action_space=self.action_space,
```

Comment:
i think you only need a single underscore? why double?

Reply:
Copied from garage_env.py. Found that `__spec` is only accessed as a private variable inside the class. Will change this in `BulletEnv` and `GarageEnv`.